Tavisca.Bootcamp.LanguageBasics.Exercise2/Program.cs

```diff
@@ -13,18 +13,63 @@ static void Main(string[] args)
             Console.ReadKey(true);
         }
 
-        private static void Test(string[] postTimes, string[] showTimes, string expected)
-        {
+        private static void Test(string[] postTimes, string[] showTimes, string expected)
+        {
             var result = GetCurrentTime(postTimes, showTimes).Equals(expected) ? "PASS" : "FAIL";
             var postTimesCsv = string.Join(", ", postTimes);
             var showTimesCsv = string.Join(", ", showTimes);
-            Console.WriteLine($"[{postTimesCsv}], [{showTimesCsv}] => {result}");
+            Console.WriteLine($"[{postTimesCsv}], [{showTimesCsv}] => {result}");
         }
 
-        public static string GetCurrentTime(string[] exactPostTime, string[] showPostTime)
-        {
-            // Add your code here.
-            throw new NotImplementedException();
+        public static string GetCurrentTime(string[] exactPostTime, string[] showPostTime)
+        {
+            var length = exactPostTime.Length;
+            List<string> array = new List<string>();
+
+            for(var i=0;i<length;i++){
+                // Check whether for same exactPostTime we have different showPostTime
+
+                if(i!=length-1){
+                    if(exactPostTime[i]==exactPostTime[i+1] && showPostTime[i]!=showPostTime[i+1])
+                        return "impossible";
+                }
+
+                 var postTime = TimeSpan.Parse(exactPostTime[i]);
+
+                // add seconds in exactPostTime
+                if(showPostTime[i].Contains("seconds")){
+                    var currentTime = new TimeSpan(postTime.Hours,postTime.Minutes,postTime.Seconds);
+                    array.Add(Convert.ToString(currentTime));
+                     
+                }
+
+                // else add minutes in exactPostTime

```

Comment:
You don't need to add else in comment..  Just say "Add Minutes to Exact Post Time"
Also comment should be normal statements, you don't need to add exact variable name in comments.
Comment should tell what a piece of code is going to do not how it is going to do it